src/opa.js

```diff
@@ -216,13 +302,19 @@ class LoadedPolicy {
   }
 }
 
-module.exports = class Rego {
-  // load_policy can take in either an ArrayBuffer or WebAssembly.Module
-  // and will return a LoadedPolicy object which can be used to evaluate
-  // the policy.
-  async load_policy(wasm) {
+class Rego {
+  /**
+   * Takes in either an ArrayBuffer or WebAssembly.Module
+   * and will return a LoadedPolicy object which can be used to evaluate
+   * the policy.
+   * @param {BufferSource | WebAssembly.Module} regoWasm
+   */
+  async loadPolicy(regoWasm) {
     const memory = new WebAssembly.Memory({ initial: 5 });
-    const policy = await _load_policy(wasm, memory);
+    const policy = await _loadPolicy(regoWasm, memory);
     return new LoadedPolicy(policy, memory);
   }
-};
+}
+
+module.exports = Rego;
+module.exports.default = Rego;
```

Comment:
Added a default export since that's more natural for ES Module users.

Reply:
👍 Sounds good to me.

nit: Can you re-add the trailing newline for the file?